docs/concepts/what-is-an-event.md

```diff
@@ -7,65 +7,144 @@ tags:
   - explanation
 ---
 
-Events are created by the Temporal Cluster in response to external occurrences and Commands generated by a Workflow Execution.
+Events are created by the Temporal Cluster in response to external occurrences and Commands generated by a Workflow Execution. Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
 
 All Events are recorded in the [Event History](/docs/concepts/what-is-an-event-history).
 
-The following is a complete list of all possible Events that could appear in a Workflow Execution Event History.
-
-Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
+A list of all possible Events that could appear in a Workflow Execution Event History is provided below.
 
 ### WorkflowExecutionStarted
 
-This is always the first Event in a Workflow Execution History and it indicates that the execution was invoked.
+This is always the first Event in a Workflow Execution History. It indicates that the execution was invoked.
+
+| Field | Type | Definition |
+| ----- | ---- | ---------- |
 
 ### WorkflowExecutionCompleted
 
-This event type indicates that the Workflow Execution has successfully completed and contains Workflow Execution results.
+This indicates that the Workflow Execution has successfully completed. The Event contains Workflow Execution results.
+
+| Field                            | Type     | Definition                                                                        |
+| -------------------------------- | -------- | --------------------------------------------------------------------------------- |
+| result                           | Payloads | Serialized result of completed Workflow.                                          |
```

Comment:
OK, so I know this is annoying to change, but I think we should drop the Type column altogether.

I think we should link to the source file in Github for more information.
But since each language converts these protobuf definitions into a language specific type - its kind of too much - and maybe confusing.

Most people will interact with Event History fields via JSON or the WebUI.